hedera-node/hedera-file-service-impl/src/main/java/com/hedera/node/app/service/file/impl/ReadableFileStoreImpl.java

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright (C) 2023 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package com.hedera.node.app.service.file.impl;
+
+import static com.hedera.node.app.service.file.impl.FileServiceImpl.BLOBS_KEY;
+
+import static java.util.Objects.requireNonNull;
+
+import com.hedera.hapi.node.base.FileID;
+import com.hedera.hapi.node.state.file.File;
+import com.hedera.node.app.service.file.FileMetadata;
+import com.hedera.node.app.service.file.ReadableFileStore;
+import com.hedera.node.app.service.mono.utils.EntityNum;
+import com.hedera.node.app.spi.state.ReadableKVState;
+import com.hedera.node.app.spi.state.ReadableStates;
+
+import edu.umd.cs.findbugs.annotations.NonNull;
+import edu.umd.cs.findbugs.annotations.Nullable;
+
+import java.util.Objects;
+import java.util.Optional;
+
+/**
+ * Provides read-only methods for interacting with the underlying data storage mechanisms for
+ * working with Topics.
```

Comment:
This looks like copy-pasta description from the TopicsStoreImpl. Update it to for FileStoreImpl

Reply:
Thanks missed it and changed to Files